libraries/ESP32/examples/DeepSleep/TouchWakeUp/TouchWakeUp.ino

```diff
@@ -74,11 +87,8 @@ void setup(){
   print_wakeup_reason();
   print_wakeup_touchpad();
 
-  //Setup interrupt on Touch Pad 3 (GPIO15)
-  touchAttachInterrupt(T3, callback, Threshold);
-
-  //Configure Touchpad as wakeup source
-  esp_sleep_enable_touchpad_wakeup();
+  //Setup sleep wakeup on Touch Pad 3 (GPIO15 for ESP32) / (GPIO3 for ESP32-S2 and S3)
+  touchSleepWakeUpEnable(T3,THRESHOLD); 
```

Comment:
I have tested it adding 2 touch pin to the sketch:

```touchSleepWakeUpEnable(T2,THRESHOLD);
touchSleepWakeUpEnable(T7,THRESHOLD);```

ESP32 wakes up with both and reports the right pin.
S2/S3 only wakes up with the last Touch Pad used in `touchSleepWakeUpEnable()`, in the example above, it means the the SoC only wakes up when touching GPIO7.

Reply:
Do we need to add more pins for wakeup to the example?

Reply:
Maybe... when it is an ESP32 only. Just to demonstrate it?! 
Also, maybe, add commentaries about the limitation for the S2/S3 in the sketch?
Just a few suggestions.

Reply:
Added both :) 
- Demontration for ESP32
- Comment in header about limitation